requirements-docs.txt

```diff
@@ -1 +1,3 @@
 Sphinx==1.3.1
+sphinx-django-command
```

Comment:
:smile: 
I created sphinx-django-command especially for improve DBBackup docs.
And other projects, of course.